forge-gui/res/cardsfolder/upcoming/the_cyber_controller.txt

```diff
@@ -0,0 +1,13 @@
+Name:The Cyber-Controller
+ManaCost:X U U B
+Types:Legendary Artifact Creature Cyberman
+PT:3/3
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigMill1 | TriggerDescription$ When CARDNAME enters the battlefield, each opponent mills X cards. Put all creature cards milled this way onto the battlefield face down under your control. They're 2/2 Cyberman artifact creatures.
+SVar:TrigMill1:DB$ Mill | Defined$ Opponent | NumCards$ X | RememberMilled$ True | SubAbility$ DBChangeZoneAll
+SVar:DBChangeZoneAll:DB$ ChangeZoneAll | Origin$ Graveyard,Exile | GainControl$ True | Destination$ Battlefield | ChangeType$ Card.Creature+IsRemembered | FaceDown$ True | FaceDownPower$ 2 | FaceDownToughness$ 2 | FaceDownSetType$ Artifact & Creature & Cyberman | SubAbility$ DBCleanup
```

Comment:
@tool4ever shouldn't be `Defined$ Remembered` be better?

Reply:
I think so... would need to be `DB$ ChangeZone` instead though